src/containers/FearConditioningContainer.tsx

```diff
@@ -75,12 +75,13 @@ export const FearConditioningContainer: ExperimentModule<FearConditioningModuleS
   useEffect(() => {
     const showVolumeToast = (volume: number) => {
```

Comment:
**Question:** Not completely familiar with the code base yet, and was just wondering if any timers need to be changed around this?

Reply:
Nah we're all good for this one, We don't need to pause the timers for this because meant to show in the background :)